lib/knapsack_pro/config/env.rb

```diff
@@ -293,6 +280,17 @@ def ci?
         def required_env(env_name)
           ENV[env_name] || raise("Missing environment variable #{env_name}")
         end
+
+        def knapsack_or_ci_env(knapsack_env_name, ci_env_name)
+          knapsack_env_value = ENV[knapsack_env_name]
+          ci_env_value = ci_env_for(ci_env_name)
+
+          if knapsack_env_value && ci_env_value
+            KnapsackPro.logger.warn("You have set the environment variable #{knapsack_env_name} which can be automatically determined from the CI environment.")
+          end
+
+          knapsack_env_value || ci_env_value
```

Comment:
Maybe I'm too paranoid, but I'd suggest:

```suggestion
          knapsack_env_value != nil ? knapsack_env_value : ci_env_value
```